File: Server/app/logging/LogManager.scala
package logging

/**
 * Created by don on 1/24/15.
 */
object LogManager {

  private var LogTargets:List[ILogTarget] = Nil

  def Create(name:String) : Logger = {
    return new Logger(name, LogTargets)
  }
  def AttachTarget(target:ILogTarget) = {
    LogTargets = LogTargets ::: (target :: Nil)
    Console.println("ATTACHING TARGET, new length %d".format(
      LogTargets.count(p=>true)
    ))

  }
}

File: Client/BEEP/Common/Logging/LogManager.cs
using System;

namespace Common.Logging
{
	public static class LogManager
	{
		private static ILogTarget[] targets;

		static LogManager ()
		{
			targets = new ILogTarget[]{new ConsoleTarget()};
		}

		public static Logger Create(object owner)
		{
			return new Logger (owner.GetType ().Name, targets);
		}

		public static Logger Create(string name)
		{
			return new Logger (name, targets);
		}
	}
}

